apps/nycaspstatus/nyc_asp_status.star

```diff
@@ -0,0 +1,117 @@
+"""
+Applet: NYC ASP Status
+Summary: Daily NYC ASP status update
+Description: Shows if New York City alternate side parking (street cleaning) rules are in effect for today (and next day after 3PM).
+Author: Adam Wojciechowski
+"""
+
+load("encoding/base64.star", "base64")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+months = ["", "Jan.", "Feb.", "Mar.", "Apr.", "May", "Jun.", "Jul.", "Aug.", "Sep.", "Oct.", "Nov.", "Dec."]
+days = ["", "1st", "2nd", "3rd", "4th", "5th", "6th", "7th", "8th", "9th", "10th", "11th", "12th", "13th", "14th", "15th", "16th", "17th", "18th", "19th", "20th", "21st", "22nd", "23rd", "24th", "25th", "26th", "27th", "28th", "29th", "30th", "31st"]
+
+URL = "https://api.nyc.gov/public/api/GetCalendar?"
+API_KEY = "5e5d2e9e95d64fcfa0be86ac934ae41f"
```

Comment:
Since this is an API key does it make sense to encrypt it with the secrets module so it's not here in plain text?